projectDocs/dev/githubPullRequestTemplateExplanationAndExamples.md

```diff
@@ -45,26 +51,26 @@ It may be worth including a summary of the history of the technical changes, and
 Example: a new app module was created, which handles an event raised by UIA.
 
 ### Testing strategy:
+
 Outline the steps you took to test the change.
 This should allow someone else to reproduce your testing.
 
 More broadly, try to answer the following questions:
+
 - How have you tested to ensure that your change works as intended?
 - Have you ensured testing coverage across all supported operating systems?
 - Have you considered possible regressions (related features or behaviours that may break)?
 
-Please use this section as an opportunity to try to convince us (and yourself) that your proposed
-change should be merged.
+Please use this section as an opportunity to try to convince us (and yourself) that your proposed change should be merged.
 
-Often in face to face development it's useful to demonstrate a change, quite often bugs are noticed
-at this point when the new person asks for some variation in testing approach.
-Since we unlikely to be able to demonstrate a feature in an interactive way, an easy-to-follow list
-of steps for a "demo" allows others check themselves without having to work out all the details.
-It also serves as a starting point for members of the community who are testing the changes that go
-into NVDA.
+Often in face to face development it's useful to demonstrate a change, quite often bugs are noticed at this point when the new person asks for some variation in testing approach.
+Since we unlikely to be able to demonstrate a feature in an interactive way, an easy-to-follow list of steps for a "demo" allows others to check for themselves without having to work out all the details.
```

Comment:
```suggestion
Since we are unlikely to be able to demonstrate a feature in an interactive way, an easy-to-follow list of steps for a "demo" allows others to check for themselves without having to work out all the details.
```